Objects/complexobject.c

```diff
@@ -905,6 +905,67 @@ complex_subtype_from_string(PyTypeObject *type, PyObject *v)
     return result;
 }
 
+static PyObject *
```

Comment:
Would it be worth adding a comment above this function explaining the purpose (and explaining why this is different from `complex_new`)?